contao/config/event_listeners.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * The MetaModels extension allows the creation of multiple collections of custom items,
+ * each with its own unique set of selectable attributes, with attribute extendability.
+ * The Front-End modules allow you to build powerful listing and filtering of the
+ * data in each collection.
+ *
+ * PHP version 5
+ *
+ * @package    MetaModels
+ * @subpackage AttributeText
+ * @author     Christopher Boelter <christopher@boelter.eu>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+use MetaModels\Attribute\Events\CreateAttributeFactoryEvent;
+use MetaModels\Attribute\Text\AttributeTypeFactory;
+use MetaModels\MetaModelsEvents;
+use MetaModels\Events\MetaModelsBootEvent;
+use MetaModels\DcGeneral\Events\Table\InputScreens\BackendSubscriber;
```

Comment:
@discordier Ok, I see the problem. but I think the namespace ```MetaModels\Events\Attribute\Text``` is not the right place for that class. It is an class that handles Events from the InputScreens-DCA so I think it is the right namespace. I think it would be better if we rename the class from ```MetaModels\DcGeneral\Events\Table\InputScreens\BackendSubscriber```to ```MetaModels\DcGeneral\Events\Table\InputScreens\TextBackendSubscriber``` so it is unique to the Attribute and there are no conflicts with the core. What do you think?